gradle/verification-metadata.xml

```diff
@@ -0,0 +1,3275 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata xmlns="https://schema.gradle.org/dependency-verification" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://schema.gradle.org/dependency-verification https://schema.gradle.org/dependency-verification/dependency-verification-1.2.xsd">
+   <configuration>
+      <verify-metadata>true</verify-metadata>
+      <verify-signatures>true</verify-signatures>
+      <trusted-artifacts>
+         <trust group="com.android.tools" name="play-sdk-proto" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.build" name="aapt2" version="8.2.0-10154469" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.external.com-intellij" name="intellij-core" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.external.com-intellij" name="kotlin-compiler" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.external.org-jetbrains" name="uast" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.lint" name="lint" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.lint" name="lint-api" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.lint" name="lint-checks" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="com.android.tools.lint" name="lint-gradle" version="31.2.0" reason="Broken: checksum is missing from verification metadata"/>
+         <trust group="org.apache.httpcomponents" name="httpclient" version="4.5.6" reason="Broken: artifact was signed but all keys were ignored, checksum is missing from verification metadata"/>
+         <trust group="org.jetbrains.kotlin" name="kotlin-stdlib" version="1.9.0" reason="Broken: artifact was signed but all keys were ignored, checksum is missing from verification metadata"/>
+         <trust group="org.jetbrains.kotlin" name="kotlin-stdlib-common" version="1.9.0" reason="Broken: artifact was signed but all keys were ignored, checksum is missing from verification metadata"/>
+      </trusted-artifacts>
+      <ignored-keys>
+         <ignored-key id="a41f13c999945293" reason="Key couldn't be downloaded from any key server"/>
```

Comment:
some keys aren't available via any dependency repo, so they will get added to an ignore list automatically